include/sparrow/allocator.hpp

```diff
@@ -53,6 +53,15 @@ namespace sparrow
                                     && (std::same_as<A, std::allocator<T>>
                                         || std::same_as<A, std::pmr::polymorphic_allocator<T>>);
 
+    template <class... Ts>
+    struct overloaded : Ts...
+    {
+        using Ts::operator()...;
+    };
+    // Although not required in C++20, clang needs it to build the code below
```

Comment:
Note that this code has been moved but the comemnts refers to "the code below".
1. Why was the move necessary?
2. Should the comment be updated?

Reply:
The issue was that the `overloaded` definition must not be in a class definition (`any_allocator`).